scripts/envDFL.sh

```diff
@@ -273,17 +281,34 @@ build_tests_coverage() {
 
 launch() {
     if [ ! -f $1 ]; then
-        error_exit "IIDM network file $network doesn't exist"
```

Comment:
This should be done in another PR because it is unrelated to SA development. Could you reset it, I'll do the corresponding PR

Reply:
ok